README.md

```diff
@@ -14,17 +14,59 @@
 
 ---
 
-The 1Password Go SDK offers programmatic access to your secrets in 1Password with Go. During the beta, you can create, retrieve, update, and delete items and resolve secret references.
-
-1Password SDKs support authentication with [1Password Service Accounts](https://developer.1password.com/docs/service-accounts/get-started/). 
-
-## ❗ Limitations
-
-1Password SDKs don't yet support using secret references with query parameters, so you can't retrieve file attachments or SSH keys, or get more information about field metadata.
-
-1Password SDKs currently only support operations on text and concealed fields. As a result, you can't edit items that include information saved in other types of fields.
-
-When managing items with 1Password SDKs, you must use [unique identifiers (IDs)](https://developer.1password.com/docs/sdks/concepts#unique-identifiers) in place of vault, item, and field names.
+## Supported Functionality
+
+1Password SDKs are in active development. We're keen to hear what you'd like to see next. Let us know by [upvoting](https://github.com/1Password/onepassword-sdk-go/issues) or [filing](https://github.com/1Password/onepassword-sdk-go/issues/new/choose) an issue.
+
+### Item Management
+Operations:
+- [x] [Retrieve secrets](https://developer.1password.com/docs/sdks/load-secrets)
+- [x] [Retrieve Items](https://developer.1password.com/docs/sdks/manage-items#get-an-item)
+- [x] [Create Items](https://developer.1password.com/docs/sdks/manage-items#create-an-item)
+- [x] [Update Items](https://developer.1password.com/docs/sdks/manage-items#edit-an-item)
+- [x] [Delete Items](https://developer.1password.com/docs/sdks/manage-items#delete-an-item)
+- [ ] List Items
+- [ ] Add & Update tags on items ([#86](https://github.com/1Password/onepassword-sdk-go/issues/86))
+
+Item/field Types:
+- [x] API Keys
+- [x] Passwords
+- [x] Logins
+- [x] Notes
+- [x] SSH Private Keys
```

Comment:
I don't think we should tick this as it's only supported in secret references. It's not supported in item create/get/update.

Reply:
```suggestion
- [x] SSH Private Keys (partially supported: supported in resolving secret references, not yet supported in item create/get/update)
```

Reply:
@Marton6 I propose we document exactly that. So we achieve both:
1. Folks who just need to reach the private keys know they can achieve this through secret references, and don't unnecessarily get disappointed with not being able to do so and needing to explore alternative options.
2. Folks who need to create or edit SSH keys, or fetch the full item, know it's not yet possible.

Reply:
Addressed in https://github.com/1Password/onepassword-sdk-go/pull/118/commits/53cfdf3498f48f4d2c0337d1586958fffd763aae. WDYT?

Reply:
I double checked how well SSH keys are supported in secret references and only retrieving the private key is supported. Other fields such as the public key, key type and fingerprint are not supported.

Reply:
Thanks for checking!